app/templates/components/paper-autocomplete.hbs

```diff
@@ -34,7 +34,7 @@
       listboxId=(readonly optionsId)
       onFocus=(action "onFocus")
       activate=(action "activate")
-      onBlur=(action "deactivate")
```

Comment:
this means we're no longer calling `deactivate`? Can't that be problematic since eps does call it?